packages/frontend/editor-ui/src/components/ResourceLocator/ResourceLocator.vue

```diff
@@ -346,6 +347,73 @@ const showOverrideButton = computed(
 	() => canBeContentOverride.value && !isContentOverride.value && !props.isReadOnly,
 );
 
+const allowNewResources = computed(() => {
+	if (!props.node) {
+		return undefined;
+	}
+
+	const addNewResourceOptions = getPropertyArgument(
+		currentMode.value,
+		'allowNewResource',
+	) as INodePropertyModeTypeOptions['allowNewResource'];
```

Comment:
There is an eslint warning to remove this `as` assertion. Do you want to remove it?